docs/community/developers.md

```diff
@@ -71,6 +71,24 @@ To set the node IP, run:
 $ export KRUSTLET_NODE_IP=<the ip address>
 ```
 
+## Testing
+
+Krustlet contains both integration and unit tests. For convenience, there are `just` targets for running one or the other.
+
+For unit tests:
+
+```console
+$ just test
+```
+
+For the integration tests, start a wascc and wasi node in separate terminals before running the tests:
+
+```console
+$ just run-wascc
```

Comment:
Nit: I would add comments like `# run in terminal 1` to each of these just to be really clear